README.md

```diff
@@ -35,6 +35,22 @@ should submit the patch on their own behalf.
 
 [The full list of individuals is available here.](individuals.md)
 
+## Howto:
+
+Issue a pull request:
+
+- include an edit to the `/individuals.md` file (put your name in alphabetic
+order) to include your own entry
+- include a new file in `/individuals/` (use [individuals/template.md](individuals/template.md))
+- format the commit comment as `individual: Add Newname` (prefix lowercase,
+colon, the rest capitalized)
+- indicate how you fare with regard to the points in the comments to the pull
+request (a template should be available soon)
```

Comment:
don't mention available soon. we'll merge that shortly =D
you can rephrase and simply this as: "fill out the self assessment in the pull request"